### ISSUE TITLE
R Studio Fails

### ISSUE THREAD

**Issue by joesoap99** [ContinuumIO/anaconda-issues#1406]
Windows 8.1, 64 bit
Anaconda 4.30 (sometimes calls it 4.38??)
Root Python 2.7.13
Environment Python 3.51

Rstudio eventually installed.
On running Rstudio from icon in Navigator it gave the error message below.
The Rstudio window was opened but none of the menus functioned:

![rstudio](https://cloud.githubusercontent.com/assets/8518350/22548555/d75111c4-e93e-11e6-964f-25529828963a.png)

**Comment by mingwandroid** [comment]
Did you click the "Launch rstudio" button multiple times while waiting for it to load?

I cannot reproduce this, can you try again with R 3.3.2 please?

**Comment by joesoap99** [comment]
Of couse I did not press the Launch button multiple times.
It is no surprise that my set up cannot be reproduced as Anaconda does not ensure cleaned up installs which are not individual install history dependent.
I have never got any upgrades to work and have had to uninstall/reinstall each time. 
It now works with Python 3.51 and R 3.2.4

Rstudio is still offered in Navigator for Python 2.7 install.
I tried to do that first and it may have upset things for the Python 3.5 install.
I was later told that RStudio does not work with Python 2.7.
